### ISSUE TITLE
etesync-dav.exe flagged as malicious by VirusTotal and blocked by SmartScreen

### ISSUE THREAD

**Issue by PhysicsIsAwesome** [etesync/etesync-dav#259]
Hi,

etesync-dav.exe is flagged as malicious on VirusTotal by 5 AV vendors, blocked by SmartScreen and the extecutable doesn't seem to be signed.

![image](https://user-images.githubusercontent.com/69597878/158864702-2838e602-5f9c-489c-9366-ea4a39e596d5.png)

Best regards,
PhysicsIsAwesome

**Comment by tasn** [comment]
It happens every now and then, there are other tickets. I don't know what to do, it's a false-positive 99%. Often trying a different version makes it detect differently in different scanners. Anti-viruses are weird. :|

**Comment by tasn** [comment]
It happens every now and then, there are other tickets. I don't know what to do, it's a false-positive 99%. Often trying a different version makes it detect differently in different scanners. Anti-viruses are weird. :|

**Comment by PhysicsIsAwesome** [comment]
I can't really help you with that. Don't know enough about AV's. Maybe you can contact these AV vendors and ask them. 

Running unsigned executables is often considered bad practice and a lot of organizations won't allow running unsigned executables on their systems. So this is a major problem and signing them would help.

**Comment by tasn** [comment]
Yeah, that I know. We definitely need to do that, it's just been a pain whenever i tried it. :|
